src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackV2VolumePluginTest.java

```
@@ -139,21 +156,29 @@ public void testGetInstance() throws IOException {
         this.expectedRequest = new HttpUriRequestMatcher(request, null);
 
         try {
+            // exercise
```

why this empty catch block?